admin/AdminPackage.php

```diff
@@ -98,13 +98,20 @@ public function error(
       return;
     }
 
+    //if there's already content
+    if ($response->hasContent()) {
+      return;
+    }
+
     //get the path
     $path = $request->getPath('string');
     //if not an admin path
     if ($path !== '/admin' && strpos($path, '/admin/') !== 0) {
       return;
     }
 
+    $debug = strpos($path, '/admin/spa/') === false ? 'page': 'spa';
```

Comment:
should we make this into constants, or is this okay since something we won't need to look back at?

Reply:
oh yea your right! nice catch. Totally forgot 👍

Reply:
ill make a note na lang to go through the packages and declare things that should be constant.

Reply:
i'll do the same if ever i run to some. boy scout mode haha